src/system/ZAuthModule/Form/Type/BatchForcePasswordChangeType.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Zikula package.
+ *
+ * Copyright Zikula Foundation - https://ziku.la/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zikula\ZAuthModule\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Zikula\GroupsModule\Entity\RepositoryInterface\GroupRepositoryInterface;
+
+class BatchForcePasswordChangeType extends AbstractType
+{
+
+    /**
+     * @var GroupRepositoryInterface
+     */
+    private $groupRepository;
+
+    public function __construct(GroupRepositoryInterface $groupRepository)
+    {
+        $this->groupRepository = $groupRepository;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('group', ChoiceType::class, [
+                'label' => 'Group to modify',
+                'required' => true,
+                'choices' => $this->getChoices(),
```

Comment:
This needs to be `'choices' => /** @Ignore */$this->getChoices(),` with `use Translation\Extractor\Annotation\Ignore;`